projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -264,6 +264,30 @@ export const getTypeDefinitionAtPosition =
         if (definition.kind.toString() === 'method') {
             return getMethodTypeDefinition(langServ, definition);
         }
+        if (entryPath.endsWith('.ts')) {
+            // for ts files we can use the type checker to look up a specific node
+            // and attempt to resolve its actual type
+            const sourceFile = langServ.getProgram().getSourceFile(entryPath);
+            const typeChecker = langServ.getProgram().getTypeChecker();
+            // const node = (tss as any).getTouchingPropertyName(sourceFile, position); -> tss internal that looks up a node
+            const node = findNodeAtPosition(sourceFile, position);
+            const symbolFromNode = typeChecker.getSymbolAtLocation(node)
+            const nodeType = typeChecker.getTypeOfSymbolAtLocation(symbolFromNode, node)
```

Comment:
PS: There's also `getTypeAtLocation` that only uses the node, w/o a symbol btw. Not sure why this API needs both, but it hints to me maybe it can do something we don't necessarily expect